src/utils/isJSON.ts

```diff
@@ -1,50 +1,46 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
 // Ported to TypeScript from:
 // https://github.com/joaquimserafim/isJSON
 
-function isString(x: string) {
+function isString(x: unknown): x is string {
   return Object.prototype.toString.call(x) === '[object String]'
 }
 
-function isObject(obj: {}) {
+function isObject(obj: unknown): obj is object {
   return Object.prototype.toString.call(obj) === '[object Object]'
 }
 
-export function isJSON(str: any, passObject?: boolean): boolean {
-  if (passObject && isObject(str)) {
+export function isJSON(value: unknown, passObject: boolean = false): boolean {
+  if (passObject && isObject(value)) {
     return true
   }
 
-  if (!isString(str)) {
+  if (!isString(value)) {
     return false
   }
 
-  str = str.replace(/\s/g, '').replace(/\n|\r/, '')
+  const str = value.replace(/\s/g, '').replace(/\n|\r/, '')
```

Comment:
One thing I didn't note in the commit message is that I changed this to avoid reassigning a parameter.